docs/rancher/03-setup-rancher/disaster-recovery-rancher.md

```diff
@@ -0,0 +1,147 @@
+# Rancher Disaster Recovery
+
+## Introduction
+
+This document offers a detailed guide on how to restore your Rancher deployment to a new cluster if the original node running Rancher fails.
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Step 1 - Switch to the k3s cluster](#step-1---switch-to-the-k3s-cluster)
+- [Step 2 - Install the rancher-backup Helm chart](#step-2---install-the-rancher-backup-helm-chart)
+- [Step 3 - Restore from backup using a Restore custom resource](#step-3---restore-from-backup-using-a-restore-custom-resource)
+- [Step 4 - Follow the Setting Up Rancher on k3s Guide](#step-4---follow-the-setting-up-rancher-on-k3s-guide)
+- [Conclusion](#conclusion)
+
+## Prerequisites
+
+Before beginning the restoration process, ensure you have:
+
+- A newly set up server following the [Setup Ubuntu](../01-setup-ubuntu/README.md) guide.
+- A backup file ready for restoration. Check the DigitalOcean space.
+
+## Step 1 - Switch to the k3s cluster
+
+It's essential to ensure you're working within the right context to apply all commands to the intended **k3s cluster**.
+
+```bash
+kubectl config use-context {{k3s-murm-rancher}}
+```
+
+## Step 2 - Install the rancher-backup Helm chart
+
+### 1. Add the Helm repository
+
+```bash
+helm repo add rancher-charts https://charts.rancher.io
+helm repo update
+```
+
+### 2. Set a `CHART_VERSION` variable
+
+Refer to the [support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-8-3/) under the Rancher Apps / Cluster Tools section to determine which rancher-backup versions are compatible. For instance, if you are using Rancher version `v2.8.3`, you should set the chart version to `103.0.1+up4.0.1`.
```

Comment:
Use this link can directly jump to Rancher Apps / Cluster Tools section
https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-8-3/#el-59173